server/storage/schema/bucket.go

```diff
@@ -69,16 +69,16 @@ func (b bucket) IsSafeRangeBucket() bool { return b.safeRangeBucket }
 
 var (
 	// Pre v3.5
-	ScheduledCompactKeyName    = []byte("scheduledCompactRev")
-	FinishedCompactKeyName     = []byte("finishedCompactRev")
-	MetaConsistentIndexKeyName = []byte("consistent_index")
-	AuthEnabledKeyName         = []byte("authEnabled")
-	AuthRevisionKeyName        = []byte("authRevision")
-	// Since v3.5
-	MetaTermKeyName              = []byte("term")
-	MetaConfStateName            = []byte("confState")
+	ScheduledCompactKeyName      = []byte("scheduledCompactRev")
```

Comment:
Can you create section `Since v3.4` to match the schema?

Reply:
I've searched release-3.3 branch and compared to 3.4 and I don't think any new fields were added

Reply:
Thanks.